test/FastRouteRouterTest.php

```diff
@@ -10,6 +10,7 @@
 namespace ZendTest\Expressive\Router;
 
 use FastRoute\Dispatcher\GroupCountBased as Dispatcher;
+use FastRoute\Dispatcher\GroupPosBased;
```

Comment:
This is a bit confusing now. One is used as `Dispatcher` and the other as `GroupPosBased`. I'd rather see something like this:
```
use FastRoute\Dispatcher;

// Inside the class use
return new Dispatcher\GroupCountBased($data);
return new Dispatcher\GroupPosBased($data);
```

This way it tells you they are both dispatchers and are different.

Reply:
Hmmm... Not sure if that works though as there is also a `FastRoute\Dispatcher` interface class 😕

Reply:
Done, the Dispatcher interface can be mocked with `$this->prophesize(Dispatcher::class);`

The `GroupCountBased` isn't even used in the tests.